util/buildRelease/install.sh

```diff
@@ -349,15 +349,24 @@ fi
 
 if [ -f "bin/$CHPL_BIN_SUBDIR/chpldoc" ]
 then
-  # create a symbolic link for chpldoc
+  # create a symbolic link for chpldoc-legacy
   if [ ! -z "$PREFIX" ]
   then
-    (cd "$PREFIX/bin" && rm -f chpldoc && ln -s chpl chpldoc)
+    (cd "$PREFIX/bin" && rm -f chpldoc-legacy && ln -s chpl chpldoc-legacy)
   else
-    (cd "$DEST_DIR/bin/$CHPL_BIN_SUBDIR" && rm -f chpldoc && ln -s chpl chpldoc)
+    (cd "$DEST_DIR/bin/$CHPL_BIN_SUBDIR" && rm -f chpldoc-legacy && ln -s chpl chpldoc-legacy)
   fi
 fi
 
+# copy chpldoc
+if [ ! -z "$PREFIX" ]
+then
+  myinstallfile "bin/$CHPL_BIN_SUBDIR"/chpldoc "$PREFIX/bin"
+else
+  tmp_bin_dir="bin/$CHPL_BIN_SUBDIR"
```

Comment:
Nitpick, but why is a temporarily variable created for this string on the `else` branch, but not the `then` branch? I feel it should be both (or created before the `if` so it's in scope for both branches) or neither.